dali/imgcodec/util/output_shape.h

```diff
@@ -21,7 +21,7 @@ namespace dali {
 namespace imgcodec {
 
 template <typename OutShape>
-void OutputShape(OutShape &&out_shape,
+void DLL_PUBLIC OutputShape(OutShape &&out_shape,
```

Comment:
I don't understand it. A template shouldn't require external linkage - unless you're doing something very wrong.

Reply:
Well I don't fully understand as well, I though that you told me to do that 😛 Reverting this then.